dissect/target/plugins/os/windows/generic.py

```diff
@@ -64,48 +64,48 @@
     "filesystem/registry/commandprocautorun",
     [
         ("datetime", "ts"),
-        ("uri", "path"),
+        ("path", "path"),
     ],
 )
 
 AlternateShellRecord = UserRegistryRecordDescriptor(
     "filesystem/registry/alternateshell",
     [
         ("datetime", "ts"),
-        ("uri", "path"),
+        ("path", "path"),
     ],
 )
 
 BootShellRecord = UserRegistryRecordDescriptor(
     "filesystem/registry/bootshell",
     [
         ("datetime", "ts"),
-        ("uri", "path"),
+        ("path", "path"),
     ],
 )
 
 FileRenameOperationRecord = UserRegistryRecordDescriptor(
     "filesystem/registry/filerenameoperations",
     [
         ("datetime", "ts"),
-        ("uri", "path"),
+        ("path", "path"),
     ],
 )
 
 WinRarRecord = UserRegistryRecordDescriptor(
     "filesystem/registry/winrar",
     [
         ("datetime", "ts"),
-        ("uri", "path"),
+        ("path", "path"),
     ],
 )
 
 WinSockNamespaceProviderRecord = UserRegistryRecordDescriptor(
     "filesystem/registry/winsocknamespaceprovider",
     [
         ("datetime", "ts"),
-        ("uri", "librarypath"),
-        ("uri", "displaystring"),
+        ("path", "librarypath"),
+        ("path", "displaystring"),
```

Comment:
Should this be a path or better a string?

Reply:
String, changed after inspection.